packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -104,6 +106,14 @@ async function createServer(
   }
 }
 
+function deleteDbIfExists(): void {
+  const workspace = vscode.workspace.rootPath;
+  const dbPath = workspace + '/.sfdx/tools/apex.db';
+  if (workspace !== undefined && fs.exists(dbPath)) {
+    fs.unlinkSync(dbPath);
```

Comment:
Can you check that this works on Windows. The process has not started yet so this should work but it's best to check.